### PR TITLE
Added API1800 support for firmware drivers and LIG uplinkset

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added API1800 support for firmware drivers and LIG uplinkset

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: spec/unit/resource/api1800/c7000/lig_uplink_set.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1800::C7000::LIGUplinkSet do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API600::C7000::LIGUplinkSet' do
+    expect(described_class).to be < OneviewSDK::API600::C7000::LIGUplinkSet
```
Comment (chebroluharika) [inline spec/unit/resource/api1800/c7000/lig_uplink_set.rb:18]: Please change it to API1600 and change in description also

File: spec/unit/resource/api1800/synergy/lig_uplink_set.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1800::Synergy::LIGUplinkSet do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API600::Synergy::LIGUplinkSet' do
+    expect(described_class).to be < OneviewSDK::API600::Synergy::LIGUplinkSet
```
Comment (chebroluharika) [inline spec/unit/resource/api1800/synergy/lig_uplink_set.rb:18]: same as above

File: spec/unit/resource/api1800/c7000/lig_uplink_set.rb (hunk 1 above)
Comment (VenkateshRavula) [reply spec/unit/resource/api1800/c7000/lig_uplink_set.rb:18]: specs are written only till API600.. so inherited from that file

File: spec/unit/resource/api1800/synergy/lig_uplink_set.rb (hunk 2 above)
Comment (VenkateshRavula) [reply spec/unit/resource/api1800/synergy/lig_uplink_set.rb:18]: done

**Pull request merged**